samples/bot-conversation/python/infra/azure.bicep

```diff
@@ -3,40 +3,87 @@
 @description('Used to generate names for all resources in this file')
 param resourceBaseName string
 
-@description('Required when create Azure Bot service')
-param botAadAppClientId string
-
-param botAppDomain string
+param webAppSKU string
 
 @maxLength(42)
 param botDisplayName string
 
-param botServiceName string = resourceBaseName
-param botServiceSku string = 'F0'
+param serverfarmsName string = resourceBaseName
+param webAppName string = resourceBaseName
+param identityName string = resourceBaseName
+param location string = resourceGroup().location
 
-// Register your web service as a bot with the Bot Framework
-resource botService 'Microsoft.BotService/botServices@2021-03-01' = {
-  kind: 'azurebot'
-  location: 'global'
-  name: botServiceName
-  properties: {
-    displayName: botDisplayName
-    endpoint: 'https://${botAppDomain}/api/messages'
-    msaAppId: botAadAppClientId
-    msaAppType: 'MultiTenant'
-    msaAppTenantId: ''
-  }
+resource identity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' = {
+  location: location
+  name: identityName
+}
+
+// Compute resources for your Web App
+resource serverfarm 'Microsoft.Web/serverfarms@2021-02-01' = {
+  kind: 'app'
+  location: location
+  name: serverfarmsName
   sku: {
-    name: botServiceSku
+    name: webAppSKU
   }
 }
 
-// Connect the bot service to Microsoft Teams
-resource botServiceMsTeamsChannel 'Microsoft.BotService/botServices/channels@2021-03-01' = {
-  parent: botService
-  location: 'global'
-  name: 'MsTeamsChannel'
+// Web App that hosts your bot
+resource webApp 'Microsoft.Web/sites@2021-02-01' = {
+  kind: 'app'
+  location: location
+  name: webAppName
   properties: {
-    channelName: 'MsTeamsChannel'
+    serverFarmId: serverfarm.id
```

Comment:
Need to add the identity to this app service. Please refer to: https://github.com/OfficeDev/teams-toolkit/blob/62c234bd5d47073cce5970dc862df7a922ac6e02/templates/ts/default-bot/infra/azure.bicep#L70